dandi/dandiapi.py

```diff
@@ -98,9 +70,9 @@ def send_request(
         :param path: A string containing the path elements for this request.
             Note that the path string should not begin or end with the path  separator, '/'.
         :type path: str
-        :param parameters: A dictionary mapping strings to strings, to be used
+        :param params: A dictionary mapping strings to strings, to be used
             as the key/value pairs in the request parameters.
-        :type parameters: dict
+        :type params: dict
         :param data: A dictionary, bytes or file-like object to send in the body.
         :param files: A dictionary of 'name' => file-like-objects for multipart encoding upload.
         :type files: dict
```

Comment:
I guess we could get rid of those `:type` doc descriptions after it gets properly typed, right?

Also in other spots, I have used numpy docstrings format, not pydoc (or whatever that original is).  we better stay consistent one way or another.